tests/models/test_restore.py

```diff
@@ -273,3 +274,4 @@ def test_model_saving_loading(tmpdir):
 def test_model_pickle(tmpdir):
     model = EvalModelTemplate()
     pickle.dumps(model)
+    cloudpickle.dumps(model)
```

Comment:
Do you have a test confirming this is deserialized properly?

In OmegaConf I have tests like:

https://github.com/omry/omegaconf/blob/master/tests/test_serialization.py#L86-L95